### ISSUE TITLE
Remove post_server.py from repository once NET global path endpoint is up and running

### ISSUE THREAD

**Issue by SPDonaghy** [UBCSailbot/sailbot_workspace#284]
### Purpose

This is just a reminder to remove test/post_server.py from this repository once the NET endpoint for the global path data is ready and the file is no longer needed for tests.

### Changes

Deleting post_server.py and modifying the test file for global_path.py will resolve this issue.
